Signal/src/ViewControllers/AppSettings/OWSLinkedDevicesTableViewController.m

```diff
@@ -46,13 +46,7 @@ - (void)viewDidLoad
     self.tableView.rowHeight = UITableViewAutomaticDimension;
     self.tableView.estimatedRowHeight = 70;
 
-    // Fix a bug that only affects iOS 11.0.x and 11.1.x.
-    if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(11, 0) && !SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(11, 2)) {
-#pragma clang diagnostic push
-#pragma clang diagnostic ignored "-Wpartial-availability"
-        self.tableView.contentInsetAdjustmentBehavior = UIScrollViewContentInsetAdjustmentNever;
-#pragma clang diagnostic pop
-    }
```

Comment:
> Just checking - did you check that we have the appropriate content inset behavior?

Are you referring to this fix?

Reply:
I've DRY'd it up in a category.

Reply:
It's related to this fix, in that it's about contentInset, but I'm not sure if adding this code to the conversation details is necessary or will fix anything. 

I guess what I'm asking for is:

- have a full page of content 
- scroll all the way down.
- verify the home button isn't overlaying any content.

And to check that this holds for iOS10, iOS11.0-1, and iOS11.2

Reply:
> verify the home button isn't overlaying any content.

I think that's now true.  The problem is that now we're adding "double content inset" to account for the home view button, at least in some cases.

Reply:
Verified that content insets are appropriate on iOS 9 and iOS 11 and iPhone X.

Lots of screenshots here:
[ScrollViewInsetsViPhoneXScreenshots.zip](https://github.com/signalapp/Signal-iOS/files/1908582/ScrollViewInsetsViPhoneXScreenshots.zip)